sources/driverdog/src/main.rs

```diff
@@ -374,8 +476,9 @@ fn run() -> Result<()> {
     }
 
     match args.subcommand {
-        Subcommand::LinkModules(_) => link_modules_sets(&all_modules_sets, args.modules_set),
-        Subcommand::LoadModules(_) => load_modules_sets(&all_modules_sets, args.modules_set),
+        Subcommand::Link(_) => link_modules_sets(&all_modules_sets, args.modules_set),
+        Subcommand::Copy(_) => copy_modules_sets(&all_modules_sets, args.modules_set),
```

Comment:
This could be consolidated into the existing `Subcommand::Link` operation, where you skip the linker invocation and just copy the `.ko` to its destination.

Reply:
I think that is actually a nice improvement. It is a little strange to only copy the files rather than link them but combining the logic based upon the config provided does simplify the codebase, even if naming is a bit loose on what `link` really means here.